### PR TITLE
[RPA 857] Blog post Youtube media player

### PR DESCRIPTION
**Pull request by MRSIMM0**
## Description

Added post Youtube player:

image

To display player you have to add link on conceitful in following format: 
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.module.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    justify-content: center;
+
+    margin-block: 10px;
+
+    width: 100%;
+    height: 100%;
+}
+
+.embed{
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.module.scss:14]: image
space is beautiful

remember that

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { VFC } from 'react';
+
+import styles from './EmbeddedVideo.module.scss';
+
+interface Props{
+    url: string
+}
+
+const EmbeddedVideo: VFC<Props> = ({url}) => (
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx:9]: ({ url })

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { VFC } from 'react';
+
+import styles from './EmbeddedVideo.module.scss';
+
+interface Props{
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx:5]: Props {

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx [hunk 4]
```diff
@@ -7,13 +7,17 @@ import Image from 'next/image';
...
+import EmbeddedVideo from '../EmbeddedVideo';
 import Typography from '../Typography';
 import styles from './RichTextRenderer.module.scss';
 
+
 interface Props {
     content: ContentfulRichText;
 }
 
+const rex = new RegExp('\\$https:\\/\\/www\\.youtube\\.com\\/watch\\?v=[^$]*\\$', 'g');
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:19]: image

create a more appropriate name for the variable

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx [hunk 5]
```diff
@@ -26,7 +30,16 @@ const RichTextRenderer: VFC<Props> = ({ content }) => {
             [BLOCKS.HEADING_4]: (_, children) => <Typography font="Roboto" className={styles.h4} variant="h4">{children}</Typography>,
             [BLOCKS.HEADING_5]: (_, children) => <Typography font="Roboto" className={styles.p} variant="h5">{children}</Typography>,
             [BLOCKS.HEADING_6]: (_, children) => <Typography font="Roboto" className={styles.p} variant="h6">{children}</Typography>,
-            [BLOCKS.PARAGRAPH]: (_, children) => <Typography font="Roboto" className={styles.p} variant="body3">{children}</Typography>,
+            [BLOCKS.PARAGRAPH]: (_, children) => {
+
+                if( children.toString().match(rex)){
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:35]: sometimes space is dangerous 

if( ch -> if(ch


image

**Review by star-in-the-jar — COMMENTED** [review]
hajs się zgadza

**Review by preludium — APPROVED** [review]

**Pull request merged**